pandas/tests/indexes/multi/test_indexing.py

```diff
@@ -439,3 +439,14 @@ def test_timestamp_multiindex_indexer():
     )
     should_be = pd.Series(data=np.arange(24, len(qidx) + 24), index=qidx, name="foo")
     tm.assert_series_equal(result, should_be)
+
+
+def test_get_loc_missing_value():
```

Comment:
can you also test .get_indexer with a nan as a value (and a valid value as well).